src/styles/style.css

```diff
@@ -0,0 +1,152 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: rgb(207, 207, 207);
+}
+
+p {
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+}
+
+.like-button {
+  cursor: pointer;
+}
+
+/* .card-img-top {
+  width: 100%;
+  height: 30vw;
+  object-fit: cover;
+} */
+
+.app-wrapper {
+  height: 100%;
+  padding: 0 0 4rem;
+}
+
+.card-img-top {
+  transition: transform 0.2s; /* Animation */
+  width: 100%;
+  margin: 0 auto;
+}
+
+.card-img-top :hover {
+  transform: scale(1.5);
+}
+
+/* .navbar {
+  margin: 24px auto;
+} */
```

Comment:
### **Optional**

- [ ] Please remove all commented code from your codebase to make your code clean and easy to maintain

Reply:
All commented code have been removed from the code base